kwok/main.go

```diff
@@ -35,9 +39,14 @@ func init() {
 }
 
 func main() {
-	ctx, op := operator.NewOperator()
+	instanceTypes, err := kwok.ConstructInstanceTypes()
+	if err != nil {
+		fmt.Printf("failed to construct instance types: %v\n", err)
```

Comment:
Rather than a `fmt` statement, could we consider using our existing logger to call `Fatal`? `logging.FromContext(ctx).Fatal()`? This would have to obviously come after initializing the operator components, but I think that's a reasonable trade-off since that's still happening before actually starting the controllers